common/getpass.go

```diff
@@ -0,0 +1,46 @@
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+// GetPass attempts to read a file for a password at the supplied location.
+// If it fails, then the original supplied string will be returned to the caller.
+// A valid string will always be returned, regardless of whether an error occurred.
+func GetPass(s string, idx int) (string, error) {
```

Comment:
Since it's a utility function, I thought it would be a good idea to expose this now rather than later.  I could imagine someone running an OT node but want to use separate account for the trans....   Oh, that doesn't make sense :-P

If you don't think we'll ever need the feature, I can remove it.

Reply:
removed in eeb1a3eaf56ff60f903b8884df5f77a0c3602dcc

Reply:
I don't see much reason for it in the near future. I'm not even sure why geth or parity expose the functionality to unlock multiple accounts, do you use multiple unlocks on a single node for mining ?

Reply:
I use multiple accounts on all of my mining nodes, yes.  There are *many* reasons why a systems integrator may associate multiple accounts to a geth instance.  I'm not arguing it here for our case though - I removed it in eeb1a3eaf56ff60f903b8884df5f77a0c3602dcc